.github/workflows/main.yaml

```diff
@@ -22,58 +22,47 @@ jobs:
           - ubuntu-latest
           - macos-latest
           - windows-latest
-        python-version:
-          - '3.11'
-          - '3.12'
+        environment:
+          - py311
+          - py312
+          - py312-jax
         exclude:
           - os: macos-latest
-            python-version: 3.11
+            environment: py311
+          - os: macos-latest
+            environment: py312
+          - os: windows-latest
+            environment: py311
           - os: windows-latest
-            python-version: 3.11
+            environment: py312
+          - os: ubuntu-latest
+            environment: py312-jax
     steps:
-      - uses: actions/checkout@v3
-      - name: Install conda environment on Windows
-        if: runner.os == 'Windows'
-        uses: mamba-org/setup-micromamba@v1
-        with:
-          init-shell: powershell
-          environment-file: environment.yml
-          environment-name: gettsim
-          cache-environment: true
-          micromamba-version: 1.5.6-0
-          create-args: >-
-            python=${{ matrix.python-version }}
-      - name: Install conda environment on Unix
-        if: runner.os != 'Windows'
-        uses: mamba-org/setup-micromamba@v1
+      - uses: actions/checkout@v4
+      - uses: prefix-dev/setup-pixi@v0.8.1
         with:
-          init-shell: bash
-          environment-file: environment.yml
-          environment-name: gettsim
-          cache-environment: true
-          create-args: >-
-            python=${{ matrix.python-version }}
-            jax
-      - name: Run tests with NumPy backend on Linux
-        # Ignore MacOS for being slow -- no reason that would break if Jax works.
+          pixi-version: v0.29.0
```

Comment:
Now at 0.34 -- any way to auto-update this?

Reply:
I could not find anything, but only looked for a brief moment.